### PR TITLE
fix: Accessibility AAA issues

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -102,3 +102,8 @@ li:hover > p > a.list-anchor {
 x-search {
   visibility: hidden;
 }
+
+
+:root {
+  --docsearch-muted-color: #666c77 !important;
```
Comment (mxschmitt) [inline src/css/custom.css:108]: this is for the top right `Search` placeholder text.

File: src/config/prismLight.js [hunk 2]
```diff
@@ -11,6 +11,10 @@ const lightTheme = require('prism-react-renderer/themes/github/index.cjs.js');
 
 module.exports = {
   ...lightTheme,
+  plain: {
```
Comment (mxschmitt) [inline src/config/prismLight.js:14]: Taken from here: https://github.com/FormidableLabs/prism-react-renderer/blob/9c041dfd8580b8b71f55c2a57e61f4c63356d4f4/packages/prism-react-renderer/src/themes/github.ts#L4-L7

File: src/config/prismLight.js [hunk 3]
```diff
@@ -86,10 +90,16 @@ module.exports = {
         color: '#6F42C1',
       },
     },
+    {
```
Comment (mxschmitt) [inline src/config/prismLight.js:93]: taken from here: https://github.com/FormidableLabs/prism-react-renderer/blob/9c041dfd8580b8b71f55c2a57e61f4c63356d4f4/packages/prism-react-renderer/src/themes/github.ts#L34-L50

and modified the color.

File: src/css/custom.css [hunk 4]
```diff
@@ -26,6 +26,7 @@
   --ifm-color-primary-lighter: #1e9124;
   --ifm-color-primary-lightest: #22a428;
   --ifm-code-font-size: 95%;
+  --ifm-link-decoration: underline;
```
Comment (mxschmitt) [inline src/css/custom.css:29]: We need to either make the color bright or make the text underline as per here.

image

Making it more bright is not possible without keeping the contrast ration as per here:

image

So maybe just underline by default on light theme?

**Review by dgozman — APPROVED** [review]

File: src/css/custom.css (hunk 4 above)
Comment (yury-s) [reply src/css/custom.css:29]: I don't like the look when all links are underlined, feels much heavier than without it:

image

Without:
image

Comment (mxschmitt) [reply src/css/custom.css:29]: Yes we will revert it, the approval was for everything except for that.

Comment (mxschmitt) [inline src/css/custom.css:29]:
```suggestion
```

**Pull request merged**